practices/coordination.md

```diff
@@ -12,48 +12,86 @@ Here are the goals we optimize for in organizing our team coordination practices
+* The [Active Development Backlog](coordination:active-backlog) contains lists of **tasks** that we are currently working on. This defines 2i2c's current activity.
 
-Each project should have the following things:
+(coordination:project-backlog)=
+### Project Backlog
 
-1. **Summarize the project**. What is the overarching scope and deverables for the project? Who is it in collaboration with? How does it fit in with the overarching [strategic goals of 2i2c](../about/strategy.md)?
-2. **Define the project's key goal**. What is the one key goal to accomplish with this project?
-3. **A {term}`Source of Truth`**.  Where does project management and tracking take place? Where do we track progress? What about to-do items? Where does communication happen around this project? (e.g., in a GitHub repository, in Slack, in a Google doc).
-4. **A collection of deliverables**. What are the deliverables that must exist in order to complete the project? These can start off more abstract, but over time they should become more concrete, and ultimately, lead to actionable tasks.
+The 2i2c Projects Backlog is defined [at this GitHub Projects Board](https://github.com/orgs/2i2c-org/projects/7). This contains all of the deliverables across our projects that we wish to work on quickly. These are organized into a few columns, representing the _state_ of each deliverable:
+
+- {guilabel}`Needs Discussion/Refinement`: Deliverables that are high-priority but un-refined. Our goal should be having discussion and doing research in order to get these deliverables ready for work.
+- {guilabel}`Ready to Work`: Deliverables that are well-scoped and have a clear path forward, and are thus ready to implement. As deliverables in {guilabel}`In progress` are completed, we should replace them with deliverables from this column. Generally speaking, deliverables near the top have higher priority than those at the bottom.
+- {guilabel}`In progress`: Deliverables that we are currently working towards. This means that we should be generating [Tasks](coordination:tasks) in our [Active Development Board](https://github.com/orgs/2i2c-org/projects/5) to complete this deliverable (more on this below).
+- {guilabel}`Blocked/Waiting`: Deliverables for which we are waiting for some action that is out of our team's immediate control.
+- {guilabel}`Done`: Deliverables that have been completed. We should close these issues and celebrate the improvements that we have made!
 
 :::{note}
-Deliverables are generally written as **user stories** - that is, with a particular user in mind, and a particular outcome they want to see.
-For example, "As a 2i2c Hub User, I want to install a package on my own."
-Deliverables should be accomplishable in one or two sprints, and may have many tasks associated with them.
+In general, there should be only two or three deliverables per team member on this board.
+It should not become so heavily-populated that it is hard to keep track of deliverables!
```

Comment:
You actually built a dev project board and an org project board... I guess you only want to mention the dev one here because of the dev-focused nature of this part of doc, am I right?

Reply:
yep correct - maybe we should add a "note" about the org one as well though. I am starting off just describing the "dev workflow" but we should generalize it over time.

Reply:
I've generalized this slightly and noted the locations of both project backlogs in the latest commit